Source/RapyutaSimulationPlugins/Public/Robots/RRRobotStructs.h

```diff
@@ -901,17 +901,18 @@ struct RAPYUTASIMULATIONPLUGINS_API FRRRobotWheelProperty
         FRRRobotWheelProperty wheelProp(MoveTemp(InWheelName));
         // [Radius/Width]
         wheelProp.WheelRadius = bFrontWheel ? 13.f : (wheelProp.WheelName.Contains(TEXT("caster")) ? 9.f : 16.f);
-        wheelProp.WheelWidth = bFrontWheel ? 10.f : 20.f;
+        wheelProp.WheelWidth = bFrontWheel ? 10.f : (wheelProp.WheelName.Contains(TEXT("caster")) ? 25.f : 14.f);
```

Comment:
As we discussed, please move these structure/class/metho to RobotImporter since it is used only by them  or add comment that these are used in the future.
Since people only see this repo, it is a little bit confusing.

Reply:
So I don't remember this being settled in discussion, the very reason why we put `FRRRobotWheelProperty` here is it is the host struct to store metadata defined in URDF/SDF, which is already part of RSPlugins as defined [here](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/devel/Source/RapyutaSimulationPlugins/Public/Core/RRURDFParser.h#L50).
* As such, we could move urdf/sdf **back** to Robotimporter, but given the very fact that URDF/SDF is not only to describe robot but any mesh-based entity, not sure we should tho.
* For now `FRRRobotWheelProperty` though it's only used by RobotImporter, but all of its properties are taken from `UChaosVehicleWheel` actually, so generally users could use it for creating their own custom vehicle wheel at runtime.